subprojects/language-java/src/main/java/org/gradle/external/javadoc/StandardJavadocDocletOptions.java

```diff
@@ -195,6 +247,17 @@ public StandardJavadocDocletOptions(MinimalJavadocOptions original) {
         }
     }
 
+    /**
+     * Gets a set of all the options that are known to this class and its super class and have separate properties.
+     *
+     * @return set of property names
+     * @since 7.5
+     */
+    @Incubating
+    public Set<String> knownOptionNames() {
+        return Sets.union(knownCoreOptionNames, knownStandardOptionNames);
```

Comment:
Corresponding change to make with the change from above.
```suggestion
        return knownStandardOptionNames;
```